views/job-roles/detail.njk

```diff
@@ -0,0 +1,138 @@
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M21 13.255A23.931 23.931 0 0112 15c-3.183 0-6.22-.62-9-1.745M16 6V4a2 2 0 00-2-2h-4a2 2 0 00-2-2v2m8 0V6a2 2 0 012 2v6a2 2 0 01-2 2H8a2 2 0 01-2-2V8a2 2 0 012-2h8zM16 10h.01M12 10h.01M8 10h.01" />
+            </svg>
+            <div>
+              <div class="font-semibold">Capability</div>
+              <div class="text-lg">{{ jobRole.capability }}</div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Band Level -->
+        <div class="bg-base-200 rounded-lg p-4">
+          <div class="flex items-center gap-3">
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6 text-primary" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 12l2 2 4-4M7.835 4.697a3.42 3.42 0 001.946-.806 3.42 3.42 0 014.438 0 3.42 3.42 0 001.946.806 3.42 3.42 0 013.138 3.138 3.42 3.42 0 00.806 1.946 3.42 3.42 0 010 4.438 3.42 3.42 0 00-.806 1.946 3.42 3.42 0 01-3.138 3.138 3.42 3.42 0 00-1.946.806 3.42 3.42 0 01-4.438 0 3.42 3.42 0 00-1.946-.806 3.42 3.42 0 01-3.138-3.138 3.42 3.42 0 00-.806-1.946 3.42 3.42 0 010-4.438 3.42 3.42 0 00.806-1.946 3.42 3.42 0 013.138-3.138z" />
+            </svg>
+            <div>
+              <div class="font-semibold">Band Level</div>
+              <div class="text-lg">{{ jobRole.band }}</div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Closing Date -->
+        <div class="bg-base-200 rounded-lg p-4">
+          <div class="flex items-center gap-3">
+            <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6 text-primary" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M8 7V3a4 4 0 118 0v4m-4 6h.01M7 21h10a2 2 0 002-2V7a2 2 0 00-2-2H7a2 2 0 00-2 2v12a2 2 0 002 2z" />
+            </svg>
+            <div>
+              <div class="font-semibold">Application Deadline</div>
+              <div class="text-lg">{{ jobRole.closingDate.toLocaleDateString('en-GB', { 
+                weekday: 'long', 
+                year: 'numeric', 
+                month: 'long', 
+                day: 'numeric' 
+              }) }}</div>
+              <div class="text-sm text-base-content/70">
+                {% set daysUntilClose = ((jobRole.closingDate - new Date()) / (1000 * 60 * 60 * 24)) | round %}
+                {% if daysUntilClose > 0 %}
+                  {{ daysUntilClose }} day{{ 's' if daysUntilClose !== 1 }} remaining
+                {% else %}
+                  Application closed
+                {% endif %}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- Actions -->
+      <div class="card-actions justify-center gap-4">
+        {% set daysUntilClose = ((jobRole.closingDate - new Date()) / (1000 * 60 * 60 * 24)) | round %}
```

Comment:
Got some magic numbers here - should we handle this in a controller instead and display it from there? 

You can save things into the locals to use in the HTML. 

I know because it's dates, it's milliseconds, seconds, minutes, hours but the code should document this